tests/test_jieba.py

```diff
@@ -0,0 +1,10 @@
+# encoding=utf-8
+import unittest
+
+import jieba
+
+
+class TestJieba(unittest.TestCase):
+    def test_text_split(self):
+        sentence = "我爱北京天安门"
+        seg_list = jieba.cut(sentence)
```

Comment:
can you add an assertion for this test please.

Maybe `self.assertEqual(X, len(seg_list))` where `X` is the number of segments.